objgraph.py

```diff
@@ -54,8 +55,8 @@
 __author__ = "Marius Gedminas (marius@gedmin.as)"
 __copyright__ = "Copyright (c) 2008-2017 Marius Gedminas and contributors"
 __license__ = "MIT"
-__version__ = '3.3.1.dev0'
-__date__ = '2017-12-28'
+__version__ = '3.3.2.dev0'
+__date__ = '2018-01-08'
```

Comment:
New function are new features, so it'll be 3.4.0.dev0, according to [SemVer](https://semver.org/).